cpp/include/raft/stats/weighted_mean.cuh

```diff
@@ -102,7 +102,7 @@ void colWeightedMean(
  * @tparam layout_t Layout type of the input matrix.
  * @param[in]  handle the raft handle
  * @param[in]  data the input matrix of size nrows * ncols
- * @param[in]  weights weight of size ncols if along_row is true, else of size nrows
+ * @param[in]  weights weight of size nrows if along_row is true, else of size ncols
```

Comment:
This should be `weights weight of size ncols if along_row is true, else of size nrows`